pkg/backend/backend.go

```diff
@@ -258,8 +258,6 @@ type UpdateOptions struct {
 	AutoApprove bool
 	// SkipPreview, when true, causes the preview step to be skipped.
 	SkipPreview bool
-	// GeneratePlan when true cause plans to be generated.
```

Comment:
This wasn't used, we already have engine.UpdateOptions on this struct which has GeneratePlan on it.